etcdctl/command/member_commands.go

```diff
@@ -132,8 +130,10 @@ func actionMemberRemove(c *cli.Context) {
 	removalID := args[0]
 
 	mAPI := mustNewMembersAPI(c)
+
+	totalCtx, _ := contextWithTotalTimeout(c)
```

Comment:
For good code habit, it needs to cancel manually. See comment here: http://godoc.org/golang.org/x/net/context#WithCancel